cartoframes/utils/geom_utils.py

```diff
@@ -127,6 +127,17 @@ def _extract_srid(egeom):
         return (0, egeom)
 
 
+def encode_geometry(geom, srid=4326):
+    """Encode geometry: wkt or wkb"""
+    wkt = True
+    if isinstance(geom, shapely.geometry.base.BaseGeometry):
+        if wkt:
```

Comment:
mmm, I don't get this condition, `wkt` is always True, right?

Reply:
Sorry, I left the flag. I will split this into two functions.

I have tested both ways of uploading data:
- `EWKT`
- `EWKB hex`

There is no clear winner :S. The second one is ~5% smaller, but it is not a big win. Do you think there is an advantage in the server-side using one over the other?

Reply:
I'd vote for `EKWB hex` if it's smaller. Also I guess that format is closer to the one PostGIS uses to store geometries (if not the same) so we might be saving computing cycles as well.